### ISSUE TITLE
The `from` filed should contain a list of mailbox instead of a single mailbox

### ISSUE THREAD

**Issue by byzhangkui** [postalsys/postal-mime#35]
According [rfc5322](https://datatracker.ietf.org/doc/html/rfc5322#section-3.6.2)
```
   from            =   "From:" mailbox-list CRLF
```
`from` filed should be a mailbox list. 

```
export type Email = {
	...
	from: Address;
	...
};
```
so, `from` should be Address[] instead of Address.

**Comment by andris9** [comment]
RFC7489 allows only a single address for the From field. https://datatracker.ietf.org/doc/html/rfc7489#section-6.6.1

**Comment by andris9** [comment]
To be correct, it is not mandated to have a single From address, but it is a convention, and many servers reject messages with multiple From addresses to simplify DMARC handling, making it a de facto standard. If you need all From addresses, not just the first one, then you can get these from the `headers` structure.

**Comment by byzhangkui** [comment]
Can the 'addressParser' method be exported, so that I can easily handle the content of the 'from' field obtained from the header?

**Comment by andris9** [comment]
Yeah, its a good idea to export it. I released v2.2.4 which also exports the `addressParser` and `decodeWords` functions. See the docs here: https://github.com/postalsys/postal-mime?tab=readme-ov-file#addressparser
